optuna/integration/lightgbm_tuner/__init__.py

```diff
@@ -31,8 +31,8 @@ def train(*args: Any, **kwargs: Any) -> Any:
     _check_lightgbm_availability()
 
     auto_booster = LightGBMTuner(*args, **kwargs)
-    booster = auto_booster.run()
-    return booster
+    auto_booster.run()
+    return auto_booster.best_booster
```

Comment:
Do we need the following change to avoid the deprecation warning?
```suggestion
    return auto_booster.get_best_booster()
```

Reply:
You're absolutely right. I applied your suggestion. Thank you!
Commit: https://github.com/optuna/optuna/pull/1076/commits/9d3b6b4abeb2626c2fc32113588392d6830687c9